src/db.rs

```diff
@@ -2028,6 +2050,74 @@ mod tests {
             db.get(&key2).await.unwrap(),
```

Comment:
I think this (and line 2025) need to get updated to reader?

Reply:
updated. Thanks for caching that.